execution/execution.go

```diff
@@ -44,7 +44,6 @@ type Execution struct {
 	TaskKey           string                 `hash:"name:taskKey"`
 	Tags              []string               `hash:"name:tags"`
 	Inputs            map[string]interface{} `hash:"name:inputs"`
-	OutputKey         string                 `hash:"-"`
```

Comment:
This modification should be done in another PR.
And also, don't forget to rename `OutputData` to `Outputs`: see "update execution db" in https://forum.mesg.com/t/simplification-of-the-tasks-output/265/6